devices/Mcp23xxx/PinValuePairEnum.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections;
+
+namespace Iot.Device.Mcp23xxx
+{
+    public class PinValuePairEnum : IEnumerator
+    {
+        public Iot.Device.Mcp23xxx.PinValuePair[] _pair;
+        int currentIndex = -1;
```

Comment:
we are using explicit visibility, I guess this one is private. Also for the naming, anything private variable should start with _

Reply:
file not necessary, removed from Mcp23xxx project